### PR TITLE
Remove duplicate API docs

### LINKED ISSUE #3148: Duplicate API docs in flutter_driver API docs
From http://docs.flutter.io/flutter_driver/flutter_driver-library.html

I see:

"""
This library provides API to test Flutter applications that run on real devices and emulators.

The application run in a separate process from the test itself. If you are familiar with Selenium (web), Espresso (Android) or UI Automation (iOS), this is Flutter's version of that.

This is Flutter's version of Selenium WebDriver (generic web), Protractor (Angular), Espresso (Android) or Earl Gray (iOS).
"""

Notice how we mention Selenium, Espresso, etc twice.

### PR DESCRIPTION
**Pull request by sethladd**
Closes https://github.com/flutter/flutter/issues/3148

### CODE REVIEW

**Comment by sethladd** [comment]
cc @yjbanov

File: packages/flutter_driver/lib/flutter_driver.dart [hunk 1]
```diff
@@ -2,12 +2,10 @@
...
 // found in the LICENSE file.
 
-/// This library provides API to test Flutter applications that run on real
+/// Provides API to test Flutter applications that run on real
 /// devices and emulators.
 ///
-/// The application run in a separate process from the test itself. If you are
-/// familiar with Selenium (web), Espresso (Android) or UI Automation (iOS),
-/// this is Flutter's version of that.
+/// The application run in a separate process from the test itself.
```
Comment (yjbanov) [inline packages/flutter_driver/lib/flutter_driver.dart]: s/run/runs/

**Comment by yjbanov** [comment]
LGTM

File: packages/flutter_driver/lib/flutter_driver.dart (hunk 1 above)
Comment (sethladd) [inline packages/flutter_driver/lib/flutter_driver.dart]: Thanks. Done.

**Comment by sethladd** [comment]
too many commits. closing.

**Pull request closed without merging**